Formula/minio-mc.rb

```diff
@@ -24,8 +24,18 @@ def install
     clipath.install Dir["*"]
 
     cd clipath do
-      system "go", "build", "-o", bin/"mc"
+      if build.head?
+        system "go", "build", "-o", buildpath/"mc"
```

Comment:
We want the tagged release to show the correct version information in `mc version`. The head version will show DEVELOPMENT_GOGET.